pkg/api/unversioned/group_version.go

```diff
@@ -56,6 +56,10 @@ type GroupVersionResource struct {
 	Resource string
 }
 
+func (gvr GroupVersionResource) IsEmpty() bool {
```

Comment:
Don't we usually call these `Empty() bool`?  It looks like group_version is the only place that is doing `IsEmpty`, but that's a bigger refactor.  Open a follow up, please.